setup.py

```diff
@@ -9,6 +9,8 @@
 
 setup(
 	name='instapipeline',
+	# python_requires='>=3.6, !=3.8.*, !=3.9.*',
```

Comment:
Can you please remove this commented out line?

Reply:
Done! Thank you for catching this!